cmd/generate/config/rules/nuget.go

```diff
@@ -0,0 +1,24 @@
+package rules
+
+import (
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func Nuget() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Nuget Password",
+		RuleID:      "nuget-config-password",
+		Regex:       regexp.MustCompile(`(?i)(<add key=\"(ClearTextPassword)|(Password)\"\s*)(value=\"[A-Za-z0-9]+\"\s*/>)`),
+		SecretGroup: 2,
+		Keywords:    []string{"cleartextpassword", "password"},
+	}
+
+	// validate
+	tps := []string{
```

Comment:
can you add an fps (i.e. negative example) to ensure this doesn't flag environment variables like this:
```
<packageSourceCredentials>
    <Contoso>
        <add key="Username" value="user@contoso.com" />
        <add key="ClearTextPassword" value="%ContosoPassword%" />
    </Contoso>
    <Test_x0020_Source>
        <add key="Username" value="user" />
        <add key="ClearTextPassword" value="%TestSourcePassword%" />
    </Test_x0020_Source>
</packageSourceCredentials>
```
https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file

I know the example in the CONTRIBUTING.md doesn't show `fps`, but you can see what I'm talking about if you look at the validate function: https://github.com/zricethezav/gitleaks/blob/cd5226711335c68be1e720b318b7bc3135a30eb2/cmd/generate/config/rules/rule.go#L57